helm/cdash/values.yaml

```diff
@@ -0,0 +1,81 @@
+cdash:
+  # cdash.host is the host name for CDash.
+  host: cdash-example.local
+
+  # cdash.https controls whether the CDash external URL
+  # uses https instead of http.
+  # When using https for a production CDash server, you should
+  # also create a Kubernetes TLS secret named "<release name>-tls"
+  # https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret_tls/
+  https: true
+
+  # cdash.image is the Docker image to use for this CDash instance.
+  image: kitware/cdash:latest
```

Comment:
Shouldn't this be pinned to the app version listed in `helm/cdash/Chart.yaml`?

Reply:
Good point, I agree that we should be consistent here and that our default value should be a release (not `latest`). But I don't think you can explicitly re-use `appVersion` in `values.yaml` since neither file is a template. So for now I'll duplicate `v3.8.0` in both files. If this gets annoying to maintain, we could drop `appVersion` altogether. It's an optional field. [Helm docs](https://helm.sh/docs/topics/charts/#the-chartyaml-file) additionally have the following to say about it:

> informational, and has no impact on chart version calculations

Reply:
Having both is fine with me.  I have a list of places I edit when making a release, as well as a total number of edits I expect to make, so adding two more isn't really problematic.

Something I hadn't considered before though: I assume that bumping the app version means that I need to make a new helm release as part of the CDash release process now?

Reply:
Yes, I believe you're correct. When we cut a new version of CDash we will want to change the default image used in the Helm chart, which means a new version for the chart as well.